src/avs/AVSList.jsx

```diff
@@ -199,55 +196,55 @@ export default function AVSList() {
           </TableHeader>
           <TableBody
             emptyContent={
-              <div className="flex h-[41.8rem] flex-col items-center justify-center">
+              <div className="flex flex-col items-center justify-center">
                 <span className="text-lg text-foreground-2">
-                  No result found for {truncate(state.searchTerm ?? '')}
+                  No result found for {state.searchTerm?.substring(0, 42)}...
                 </span>
               </div>
             }
           >
             {state.isFetchingAvsData
               ? [...Array(10)].map((_, i) => (
                   <TableRow key={i} className="border-t border-outline">
-                    <TableCell className="h-[3.82rem] w-2/5">
-                      <Skeleton className="h-5 rounded-md dark:bg-default" />
+                    <TableCell className="w-2/5 px-5 py-6">
+                      <Skeleton className="h-5 rounded-md bg-default" />
```

Comment:
```suggestion
                      <Skeleton className="h-4 rounded-md bg-default" />
```
To fix the layout shift.